### ISSUE TITLE
Mod is bundling deftulib when build and the version used breaks on servers. 

### ISSUE THREAD

**Issue by AzureDoom** [Deftu/CrashBrander#2]
```
[31.10 22:25:00] [Server] java.lang.RuntimeExceptionCould not execute entrypoint stage 'main' due to errors, provided by 'deftulib'!
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.minecraft.server.Main.main(Main.java:101) [server-intermediary.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) [fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] Caused byjava.lang.RuntimeException: Cannot load class net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents in environment type SERVER
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.transformer.FabricTransformer.transform(FabricTransformer.java:59) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPreMixinClassByteArray(KnotClassDelegate.java:454) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:407) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.10.jar:?]
[31.10 22:25:00] [Server] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
[31.10 22:25:00] [Server] 	at xyz.deftu.lib.updater.UpdateChecker.start(UpdateChecker.kt:28) ~[deftulib-1.1.0-b00c19845439bc94.jar:?]
[31.10 22:25:00] [Server] 	at xyz.deftu.lib.DeftuLib.onInitialize(DeftuLib.kt:57) ~[deftulib-1.1.0-b00c19845439bc94.jar:?]
[31.10 22:25:00] [Server] 	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.10.jar:?]
```

**Comment by Deftu** [comment]
Oops, damn
I'll fix this ASAP

**Comment by Deftu** [comment]
Can't believe I let this get past me :/

**Comment by AzureDoom** [comment]
It happens! lol

**Comment by Deftu** [comment]
Still upset because it's the first bug I've pushed to prod in quite a while :P

**Comment by AzureDoom** [comment]
I'll trade you 8x Geckolib loaders broken for 4 updates. So about 32 files pushed before fixed XD

**Comment by Deftu** [comment]
Should now be fixed in a64df6241908d426b1a48886e1edf006a2fe6ced
